backend/src/stats.js

```diff
@@ -0,0 +1,98 @@
+const {
+  queryTransactionsByDate,
+  queryTeragasUsedByDate,
+  queryNewAccountsByDate,
+  queryNewContractsByDate,
+} = require("./db-utils");
+const { formatDate } = require("./utils");
+
+let TRANSACTIONS_BY_DATE = null;
+let TERAGAS_USED_BY_DATE = null;
+let NEW_ACCOUNTS_BY_DATE = null;
+let NEW_CONTRACTS_BY_DATE = null;
+
+const calculateTransactionsByDate = async function () {
```

Comment:
@khorolets Consider renaming to

```suggestion
const aggregateTransactionsCountByDate = async function () {
```